src/NUnitFramework/framework/Api/NUnitTestAssemblyRunner.cs

```diff
@@ -34,7 +34,7 @@
 using System.Security;
 using NUnit.Framework.Internal.Abstractions;
 
-#if NET35 || NET40 || NET45
+#if NET45
```

Comment:
Possibly was meant to be `#if NETFRAMEWORK` (a bunch of other places as well)

Reply:
Good point @jnm2, most instances should be NETFRAMEWORK and I've changed as appropriate.